pkg/logger/logger_test.go

```diff
@@ -47,26 +48,18 @@ func TestMessage(t *testing.T) {
 		default:
 			t.Errorf("%s: Unknown name", tc.name)
 		}
-		if logs.Len() != 1 {
-			t.Errorf("%s: Expected log length to be 1, actual %d", tc.name, logs.Len())
-		}
-		actualMessage := logs.All()[0]
-		if actualMessage.Message != testMessage {
-			t.Errorf("%s: Expected message to be %s, actual %s", tc.name, testMessage, actualMessage.LoggerName)
-		}
-		expectedZapFields := map[string]interface{}{}
-		if !reflect.DeepEqual(actualMessage.ContextMap(), expectedZapFields) {
-			t.Errorf("%s: Expected zap fields %v actual %v", tc.name, expectedZapFields, actualMessage.ContextMap())
-		}
-		if actualMessage.Level != tc.expectedLevel {
-			t.Errorf("%s: Expected message level to be %d, actual %d", tc.name, tc.expectedLevel, actualMessage.Level)
+		require.Equal(t, 1, logs.Len())
 
-		}
+		actualMessage := logs.All()[0]
+		require.Equal(t, testMessage, actualMessage.Message)
 
+		expectedZapFields := map[string]interface{}{}
+		require.True(t, reflect.DeepEqual(actualMessage.ContextMap(), expectedZapFields))
```

Comment:
We should be able to just use require.Equal.  Looking at the implementation (https://github.com/stretchr/testify/blob/c5fc9d6b6b21ea89be8480c0dc35e2977ab988f6/require/require.go#L158, https://github.com/stretchr/testify/blob/c5fc9d6b6b21ea89be8480c0dc35e2977ab988f6/assert/assertions.go#L58), it looks like under the hood it is doing deep equal.